libs/net/traffic_generator.py

```diff
@@ -18,10 +18,12 @@
 class BaseTcpClient(ABC):
     """Base abstract class for network traffic generator client."""
 
-    def __init__(self, server_ip: str, server_port: int):
+    def __init__(self, server_ip: str, server_port: int, maximum_segment_size: int | None = None):
```

Comment:
The `None` default is redundant. You can place the default to `0` and keep the type clean.